sklearn/metrics/ranking.py

```diff
@@ -47,13 +47,19 @@ def auc(x, y, reorder=False):
     Parameters
     ----------
     x : array, shape = [n]
-        x coordinates.
+        x coordinates. These must be either monotonic increasing or monotonic
+        decreasing.
     y : array, shape = [n]
         y coordinates.
```

Comment:
don't these also need to be correspondingly increasing/decreasing in case of ties?